units/host_status.go

```diff
@@ -176,27 +188,32 @@ func (j *cloudHostReadyJob) setCloudHostStatus(ctx context.Context, m cloud.Mana
 		if err := j.initialSetup(ctx, m, &h); err != nil {
 			return errors.Wrap(err, "problem doing initial setup")
 		}
-		return j.setNextState(h)
+		err := j.setNextState(h)
+		j.logHostStatusMessage(&h, cloudStatus)
+		return err
```

Comment:
should we Wrap/WithStack this error?

Reply:
I don't think it needs to be wrapped again because it's already wrapped by `setNextState`.